src/main/java/bio/terra/service/filedata/google/gcs/GcsConstants.java

```diff
@@ -0,0 +1,13 @@
+package bio.terra.service.filedata.google.gcs;
+
+public class GcsConstants {
+
+  private GcsConstants() {}
+
+  /**
+   * Query parameter on gs:// urls that can be used to specify what google project to bill when
+   * accessing the blob. Note: this is not a Google-recognized query parameter so it must be removed
+   * when accessing the file using Google clients
+   */
+  public static final String USER_PROJECT_QUERY_PARAM = "userProject";
```

Comment:
This already exists in two other places. If it's the same concept, could they all use the same constant? I see `DrsService.USER_PROJECT_QUERY_PARAM` and `BlobContainerCopier.USER_PROJECT_QUERY_PARAM`

Reply:
They're subtly different.  They happen to be the same values but one is the one formally recognized by Google and used in their APIs, the other is ours so I wanted to keep them separate